2025_rcw_hackday_tdastro_nb/model_from_prototype.ipynb

```diff
@@ -0,0 +1,1645 @@
+       "</table><table border=\"1\" class=\"dataframe\">\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td>60926.355087</td>\n",
+       "      <td>y</td>\n",
+       "      <td>559186.436298</td>\n",
+       "      <td>1296.259292</td>\n",
+       "      <td>558809.578261</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>814 rows × 5 columns</p></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "1 rows x 7 columns"
+      ],
+      "text/plain": [
+       "   id         ra        dec  nobs     z  \\\n",
+       "0   0  74.791619 -34.358861   814  None   \n",
+       "\n",
+       "                                              params  \\\n",
+       "0  {'radec.ra': 74.79161897277173, 'radec.dec': -...   \n",
+       "\n",
+       "                                          lightcurve  \n",
+       "0  [{mjd: 60926.355087, filter: 'y', flux: 559186...  "
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "simulation = simulate_lightcurves(lightcurve_source, 1, opsim_data, lsst_passbands)\n",
+    "simulation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "9e35abdc1068194f",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-07-25T17:26:36.085535Z",
+     "start_time": "2025-07-25T17:26:36.080986Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>mjd</th>\n",
+       "      <th>filter</th>\n",
+       "      <th>flux</th>\n",
+       "      <th>fluxerr</th>\n",
+       "      <th>flux_perfect</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>60926.355087</td>\n",
+       "      <td>y</td>\n",
+       "      <td>559186.436298</td>\n",
+       "      <td>1296.259292</td>\n",
+       "      <td>558809.578261</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>60926.379060</td>\n",
+       "      <td>y</td>\n",
+       "      <td>810077.638327</td>\n",
+       "      <td>1579.958076</td>\n",
+       "      <td>810967.425721</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\
```

Comment:
Add a comment that this adds magnitude and magnitude error columns to a data frame based on the corresponding flux and fluxerr columns.